channels/iq.m3u

```diff
@@ -137,3 +137,11 @@ https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ZagrosTV.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="" group-title="",Zagros TV (480p) [Not 24/7]
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/48.m3u8
+#EXTINF:-1 tvg-id="AlEtejah.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/jFKKRjP.png" group-title="News",Al-Etejah (576p)
+https://streaming.aletejahtv.iq:1937/etejah-live/smil:etejah.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlJawadain.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/tLekUZU.png" group-title="Religious",Al-Jawadain TV (720p)
+https://live.aljawadain.org/live/aljawadaintv/playlist.m3u8
```

Comment:
404 not found. Not 24/7 ?

Reply:
Must be. It played just minutes ago. 

Now I get:
https://live.aljawadain.org/live/aljawadaintv/playlist.m3u8: Server returned 404 Not Found

Tried with and without proxychains.

Reply:
Noted, i'll add a Not 24/7 bracket.